Vagrantfile

```diff
@@ -0,0 +1,36 @@
+Vagrant.configure("2") do |config|
+
+  config.vm.define "test" do |test|
+    test.vm.box = "centos/7"
+    test.vm.provider "virtualbox" do |vb|
+        vb.memory = 2048
+    end
+
+    test.vm.network "private_network", ip: "192.168.250.10"
```

Comment:
Does this need IP and hostname set? You can usually get away without that unless you explicitly need it.